repair/repair.cc

```diff
@@ -1411,6 +1502,11 @@ static int do_repair_start(seastar::sharded<database>& db, seastar::sharded<netw
     // Do it in the background.
     (void)repair_tracker().run(id, [&db, &ms, id, keyspace = std::move(keyspace),
             cfs = std::move(cfs), ranges = std::move(ranges), options = std::move(options), ignore_nodes = std::move(ignore_nodes)] () mutable {
+
+        auto participants = options.get_participating_hosts(db.local());
+        auto all_nodes = db.local().get_token_metadata().get_all_endpoints();
+        try_wait_for_hints_to_be_replayed(id, std::move(all_nodes), std::move(participants)).get();
```

Comment:
We need to exclude ignore_nodes from all_nodes too.

Reply:
OK, done.